third_party/cargo/remote/BUILD.memoffset-0.6.5.bazel

```diff
@@ -26,23 +26,47 @@ package(default_visibility = [
 ])
 
 licenses([
```

Comment:
The diff for this whole file is silly because it reflects removal of a file and creation of another with similar formatting. ::shrug::

Reply:
yeah I'm not sure why the diff rendered that way....